### PR TITLE
Added deny.toml and cargo.toml headers to code snippets

### PR DESCRIPTION
**Pull request by TuckerBMorgan**
Took a look at the docs to add the file headers for deny, and Cargo, seemed to only to be two place in a single file. Choose not to do it to every code snippet, can if that is desired. 
Issue #342

### CODE REVIEW

File: docs/src/checks/licenses/cfg.md [hunk 1]
```diff
@@ -201,13 +201,15 @@ It's often not useful or wanted to check for licenses in your own private worksp
 If `true`, workspace members will not have their license expression checked if they are not published.
 
 ```ini
+#Cargo.toml
```
Comment (Jake-Shadle) [inline docs/src/checks/licenses/cfg.md:204]:
```suggestion
# Cargo.toml
```

File: docs/src/checks/licenses/cfg.md [hunk 2]
```diff
@@ -201,13 +201,15 @@ It's often not useful or wanted to check for licenses in your own private worksp
...
 [package]
 name = "sekret"
 license = "¯\_(ツ)_/¯"
 publish = false # "private"!
 ```
 
 ```ini
+#deny.toml
```
Comment (Jake-Shadle) [inline docs/src/checks/licenses/cfg.md:212]:
```suggestion
# deny.toml
```

File: docs/src/checks/licenses/cfg.md [hunk 3]
```diff
@@ -219,12 +221,14 @@ A list of private registries you may publish your workspace crates to. If a work
 
 ```ini
 [package]
+#Cargo.toml
```
Comment (Jake-Shadle) [inline docs/src/checks/licenses/cfg.md:224]:
```suggestion
# Cargo.toml
[package]
```

File: docs/src/checks/licenses/cfg.md [hunk 4]
```diff
@@ -219,12 +221,14 @@ A list of private registries you may publish your workspace crates to. If a work
...
 name = "sekret"
 license = "¯\_(ツ)_/¯"
 publish = ["sauce"]
 ```
 
 ```ini
+#deny.toml
```
Comment (Jake-Shadle) [inline docs/src/checks/licenses/cfg.md:231]:
```suggestion
# deny.toml
```

**Review by Jake-Shadle — CHANGES_REQUESTED** [review]
This change is just cosmetic so might as well just change all of the code snippets at once. :slightly_smiling_face:

**Comment by TuckerBMorgan** [comment]
thanks for doing the other parts!

**Review by TuckerBMorgan — COMMENTED** [review]
looks good to me

**Review by Jake-Shadle — APPROVED** [review]
Thanks for the PR!

**Pull request merged**